PetAdoption-iOS/PetAdoption-iOS/Controllers/PetListingDetailVC.swift

```diff
@@ -202,6 +203,21 @@ class PetListingDetailVC: UITableViewController, MFMailComposeViewControllerDele
     func mailComposeController(_ controller: MFMailComposeViewController, didFinishWith result: MFMailComposeResult, error: Error?) {
         controller.dismiss(animated: true, completion: nil)
     }
+    
+    ////////////////////////////////////////////////////////////
+    // MARK: - Phone Logic
+    ////////////////////////////////////////////////////////////
+    
+//    func callCellPushed(sender: AnyObject) {
+//        if let url = URL(string: "tel://\(pet?.contact.phone)") {
+//            if #available(iOS 10.0, *) {
+//                UIApplication.shared.open(url, options: [:], completionHandler: nil)
+//            } else {
+//                print("Unable to send call.")
+//            }
+//        }
+//    }
+//
```

Comment:
Typically I would say not to leave commented out code like this in.  It's fine for now since I know you're working on the phone number issue next.

Reply:
Got it